cluster.js

```diff
@@ -975,6 +993,73 @@ class SyncCluster extends Cluster {
     }
 };
 
+class PrometheusCluster extends Cluster{
+    constructor(c) {
+        super({
+            name:       c.name || 'PrometheusCluster',
+            spec:       {protocol: 'Prometheus'},
+            endpoints:  [],
+        });
+        this.type = 'byte-stream'
+    }
+
+    async _stream(s){
+        log.silly('PrometheusCluster.stream:', `Session: ${s.name}`);
+
+        // the writable part is in objectMode: result is status/message
+        let passthrough =
+            new utils.DuplexPassthrough({}, {writableObjectMode: true});
+        let stream = passthrough.right;
+        // eventDebug(stream);
+        let body = '';
+        stream.on('data', (chunk) => { body += chunk; });
+        stream.on('end',  () => { this.handleRequest(s, body, stream) } );
+
+        return Promise.resolve({stream: passthrough.left,
+            endpoint: this.virtualEndPoint() });
+    }
+
+    async handleRequest(s,body,stream){
+        log.silly('l7mp.prometheus: handleRequest:', dumper(s.metadata, 10))
+
+        let merged = monitoring.mergeRegistries();
+        let exposedMetrics = await merged.metrics();
+        log.silly('l7mp.prometheus: expose custom metrics:', dumper(exposedMetrics,100));
```

Comment:
Please, make log format consistent with the rest of the code: `log.silly('PrometheousCluster.handleRequest', ...)`

Reply:
Changed it in the latest commits.